build.sbt

```diff
@@ -9,7 +9,9 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
   .settings(publishSettings)
   .settings(scalaMacroDependencies)
   .settings(moduleName := "magnolia")
-  .settings(libraryDependencies ++= Seq("com.propensive" %% "mercator" % "0.2.1"))
+  .settings(libraryDependencies ++= Seq(
+    ("com.propensive" %% "mercator" % "0.2.1").exclude("org.scala-lang", "scala-reflect")
```

Comment:
I didn't realise you could do this! Just out of interest, does having `scala-reflect` on the classpath cause any issues, or is it just JAR bloat?

Reply:
Unexpected bloat as I've been trying to keep scala-reflect out of all library dependencies and it just moved back in after adding magnolia.